src/components/pages/settings.yml

```diff
@@ -13,3 +13,6 @@ termsPageHeader:
 footerLinksHeader:
   en: Footer links
   de: Footer-Links
+defaultTagsHeader:
+  en: Default tags
+  de: Vordefinierte Tags
```

Comment:
I guess we need Marke or Schild @ahelmberger ?

Reply:
"Tag" is the word :-)